src/middleware/src/storage/src/websql.js

```diff
@@ -1,26 +1,33 @@
-import { NotFoundError } from './errors';
-import Promise from 'es6-promise';
+import { NotFoundError, isDefined } from 'kinvey-node-sdk/dist/export';
 import map from 'lodash/map';
 import forEach from 'lodash/forEach';
 import isArray from 'lodash/isArray';
 import isFunction from 'lodash/isFunction';
 import isString from 'lodash/isString';
 const idAttribute = process.env.KINVEY_ID_ATTRIBUTE || '_id';
 const masterCollectionName = 'sqlite_master';
+const size = 5 * 1000 * 1000; // Database size in bytes
 let dbCache = {};
 let isSupported;
-const SIZE = 5 * 1000 * 1000; // 5mb
 
-export default class WebSQL {
+class WebSQL {
   constructor(name = 'kinvey') {
+    if (isDefined(name) === false) {
+      throw new Error('A name is required to use the IndexedDB adapter.', name);
+    }
+
+    if (isString(name) === false) {
+      throw new Error('The name must be a string to use the IndexedDB adapter', name);
+    }
+
     this.name = name;
   }
 
   openDatabase() {
     let db = dbCache[this.name];
 
     if (!db) {
-      db = global.openDatabase(this.name, 1, '', SIZE);
+      db = global.openDatabase(this.name, 1, '', size);
```

Comment:
@thomasconner  What is the third parameter? Can we provide a meaningful value for it?